NeedABreak/SettingsWindow.xaml.cs

```diff
@@ -163,5 +164,13 @@ private void CloseButton_Click(object sender, RoutedEventArgs e)
         {
             Close();
         }
+        private void LockNow(object sender, RoutedEventArgs e)
+        {
+            System.Diagnostics.Process.Start("rundll32.exe", "user32.dll LockWorkStation");
+
+
+
+        }
+
```

Comment:
Please use LockWorkStation P/Invoke method instead of Process.Start.  You can look in MainWindow.xaml.cs. Ideally the code should be refactored have only one declaration of the method.